requestmanager/asyncloader/asyncloader.go

```diff
@@ -31,7 +31,6 @@ type alternateQueue struct {
 
 // Allocator indicates a mechanism for tracking memory used by a given peer
 type Allocator interface {
-	AllocateBlockMemory(p peer.ID, amount uint64) <-chan error
 	ReleaseBlockMemory(p peer.ID, amount uint64) error
```

Comment:
So now an `Allocator` interface only has a `ReleaseBlockMemory` method.  Is `Allocator` still an appropriate name?

Reply:
This is just that go pattern of defining the interface at the site of usage. I guess we could rename it? Not sure.